packages/lodestar/src/api/types/node.ts

```diff
@@ -11,7 +11,7 @@ export type NodeIdentity = {
 export type NodePeer = {
   peerId: string;
   enr: string;
-  address: string;
+  lastSeenP2pAddress: string;
   state: "disconnected" | "connecting" | "connected" | "disconnecting";
-  direction: "inbound" | "outbound";
+  direction: "inbound" | "outbound" | "NA";
```

Comment:
What does it mean `NA`? Unknown direction? Do you know under which conditions does it happen?

Reply:
if you are disconnected.

Reply:
Have we decided to use the string `NA` or does it come from libp2p or the specs?

Reply:
neither. It isn't covered in api spec so every client did it's own thing :(

Reply:
Then can we use the string `disconnected`?

Reply:
I would rather use null.

Reply:
Sounds good, with a comment explaining this nuance